content/20.treatments.md

```diff
@@ -617,57 +617,117 @@ Biologics are produced from components of living organisms or viruses. They incl
+Matched control refers to studies that sought to retrospectively identify pairs of comparable patients who received and didn't receive TCZ.
+PCR-confirmed indicates whether patients' COVID-19 diagnosis was confirmed by a laboratory test (RT-PCR).
+IMV = Invasive Medical Ventilation {#tbl:tcz-table}
+-->
+Many studies use a retrospective, observational design, where they compare outcomes for COVID-19 patients who received TCZ to those who did not over a set period of time.
+For example, one of the largest retrospective, observational analysis released to date [@doi:10/d2pk] compared the rates at which patients who received TCZ deceased or progressed to invasive medical ventilation over a 14-day period compared to patients receiving only SOC.
+
+Under this definition, SOC could include other drugs such as HCQ, azithromycin, lopinavir-ritonavir or darunavir-cobicistat, or heparin.
+While this study was not randomized, a subset of patients who were eligible to receive TCZ were unable to obtain it due to shortages; however, these groups were not directly compared in the analysis.
+After adjusting for variables such as age, sex, and SOFA score, they found that patients treated with TCZ were less likely to progress to invasive medical ventilation and/or death (adjusted hazard ratio = 0.61, CI 0.40-0.92, p=0.020), although analysis of death and ventilation separately suggests that this effect may have been driven by differences in the death rate (20% of control versus 7% of TCZ-treated patients).
+They reported particular benefits for patients whose P/F Ratio fell below a 150 mm Hg threshold and found no differences between groups administered subcutaneous versus intravenous TCZ.
+Another retrospective observational analysis of interest examined the charts of patients at a hospital in Connecticut, USA where 64% of all 239 COVID-19 patients in the study period were administered TCZ based on assignment by a standardized algorithm [@doi:10.1016/j.chest.2020.06.006].
+They found that survivorship was more similar in patients presenting with severe vs. nonsevere COVID-19  at intake (severe vs. nonsevere) when TCZ was administered and proposed that their algorithm was able to target TCZ to patients presenting with CRS.
+They also reported higher survivorship in Black and Hispanic patients compared to white patients when adjusted for age.
+The major limitation with interpretation for these studies is that there may be clinical characteristics that influenced medical practitioners decisions to administer TCZ to some patients and not others.
+One interesting example therefore comes from an analysis of patients at a single hospital in Brescia, Italy, where TCZ was not available for a period of time [@doi:10.1016/j.ejim.2020.05.009].
+This study compared COVID-19 patients admitted to the hospital before and after March 13, 2020, when the hospital received TCZ.
```

Comment:
Was the characteristics of the patients similar before vs after? i.e. similar age, gender, severity of symptoms?

Reply:
Great question Alex. So they didn't do statistical tests but they don't look comparable! That is a great catch. See comment below.